### PR TITLE
Add Symfony 6 compatibility

### PR DESCRIPTION
**Pull request by jdecool**
Symfony 6 has been release.

Allow to use this bundle with the latest Symfony versions.

### CODE REVIEW

**Review by lnahiro — APPROVED** [review]

**Review by bestpostmaster — APPROVED** [review]

**Review by thibaudgirard — APPROVED** [review]

File: composer.json [hunk 1]
```diff
@@ -14,13 +14,13 @@
     ],
     "require": {
         "php": ">=7.1",
-        "symfony/framework-bundle": "~4.2 || ~5.0",
+        "symfony/framework-bundle": "~4.2 || ~5.0 || ~6.0",
```
Comment (Oliboy50) [inline composer.json:17]: same [comment for very old symfony versions](https://github.com/BedrockStreaming/GuzzleHttpBundle/pull/75#discussion_r761796423) 

=> `>= 4.4` should be good enough IMO

**Review by Oliboy50 — APPROVED** [review]

File: composer.json (hunk 1 above)
Comment (jdecool) [reply composer.json]: ✅

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -4,11 +4,12 @@ on: [push]
 jobs:
   tests:
     name: Tests
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-20.04
     strategy:
       matrix:
-        version: ['7.1', 7.2', '7.3', '7.4', '8.0']
+        version: ['7.1', '7.2', '7.3', '7.4', '8.0', '8.1']
```
Comment (valentin-claras) [inline .github/workflows/ci.yml:10]: Do we need to keep unsupported versions in the tests?

**Review by valentin-claras — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (jdecool) [reply .github/workflows/ci.yml]: No, I don't think so.

Fixed in b6dbc6a

**Pull request merged**